src/android/StatusBar.java

```diff
@@ -142,6 +142,23 @@ public void run() {
             return true;
         }
 
+        if ("overlaysWebView".equals(action)) {
+            if (Build.VERSION.SDK_INT >= 21) {
```

Comment:
This one is used to check the return value of the method, the one on line 186 is used to isolate API 21+ code, so that the compiler handles it correctly and doesn't complain about it when building for older versions of Android.